lib/trade-manager/cash-token.js

```diff
@@ -33,6 +34,10 @@ class CashTokenClient extends BaseClient {
     if (!this.erc20) {
       throw new Error('Must provide the address of the ERC20 contract');
     }
+    this.erc1155 = Config.getERC1155Address();
```

Comment:
given that a test scenario will have either ERC20 or ERC1155 based cash token contracts, the check can be if either address is provided, rather than requiring both at the same time.